packages/core/EventManager.ts

```diff
@@ -44,6 +45,13 @@ interface HasId {
   id: number;
 }
 
+// The options should have the slug of the apps the option is enabled for
+interface AppOptions {
+  crm?: {
+    skipContactCreation?: string[];
+  };
```

Comment:
Break the options down into app category and the option. The option is enabled for an app if the slug is present in the array.